DESCRIPTION

```diff
@@ -43,12 +43,15 @@ Imports:
     rlang,
     smoothqr,
     stats,
+    stringr,
+    stringi,
```

Comment:
brief reason? I'll try to remove them regardless, I'm just wondering

Reply:
Brief: "fewer Imports is better"
Longer: `{stringi}` uses some beefy compiled code (even the precompiled binaries are large), and can sometimes fail to install on some systems. There are a handful of packages like this, `{sf}` is another, that I prefer to avoid, where possible.